lib/services/rpcbind.pm

```diff
@@ -52,6 +52,11 @@ sub stop_service {
     common_service_action($nfs_server, $service_type, 'stop');
 }
 
+sub is_enabled {
```

Comment:
Since this routine is not returning a boolean variable, I would rename it to `check_enabled`.
From a routine called `is_enabled` I would expect to get a true or false as a return value. `check_enabled` on the other hand suggests, that checks are performed but nothing will be returned.